fs/fs.go

```diff
@@ -78,9 +78,10 @@ func NewFsInfo(context Context) (FsInfo, error) {
 	}
 	supportedFsType := map[string]bool{
 		// all ext systems are checked through prefix.
-		"btrfs": true,
-		"xfs":   true,
-		"zfs":   true,
+		"btrfs":   true,
+		"overlay": true,
```

Comment:
I think the Q is, what api is this trying to provide, I've tried to make heads or tails out of it, and I don't know.  

i.e. why isn't doing a statfs on a given dir good enough at all times?  why do we care about the partition table?